Box.V2/Config/BoxConfig.cs

```diff
@@ -122,124 +142,124 @@ public static IBoxConfig CreateFromJsonString(string jsonString)
             return new BoxConfig(clientId, clientSecret, enterpriseId, privateKey, rsaSecret, publicKeyId);
         }
 
-        public virtual Uri BoxApiHostUri { get; set; } = new Uri(Constants.BoxApiHostUriString);
-        public virtual Uri BoxAccountApiHostUri { get; set; } = new Uri(Constants.BoxAccountApiHostUriString);
-        public virtual Uri BoxApiUri { get; set; } = new Uri(Constants.BoxApiUriString);
-        public virtual Uri BoxUploadApiUri { get; set; } = new Uri(Constants.BoxUploadApiUriString);
+        public Uri BoxApiHostUri { get; private set; } = new Uri(Constants.BoxApiHostUriString);
```

Comment:
They were not overridden anywhere in our SDK. If we expose IBoxConfig it could be used instead so there is no value for having virtual properties there (also the class is now sealed so it could not be derived from). See also explanation in #401.